.github/workflows/ci.yml

```diff
@@ -155,9 +155,9 @@ jobs:
           yarn --frozen-lockfile --ignore-scripts --prefer-offline
           yarn download-tools --target ${{ matrix.target }} --no-cache
 
-      - name: Create vsix package (pre-release)
+      - name: Create vsix package
         run: |
-          yarn package --target ${{ matrix.target }} --pre-release
+          yarn package --target ${{ matrix.target }}

```

Comment:
Do we not plan any further pre-releases anymore?
Or should this rather become a workflow parameter, so that we can decide on purpose whether to have a pre-release or a full release?

Reply:
Having a pre-release flow would become handy after the first v1.0.0 release later in the summer. So, the answer is yes. We can however live with this until that release happened. That's when we start to claim production quality.

Reply:
IIRC, we were due to make a release at the time and that was more of a hack to ensure we can push things through to a pre-release. The corresponding publish tool checks if something was built with or without this flag depending on the requested channel to publish to.